integration-tests/beanio/src/test/java/org/apache/camel/quarkus/component/beanio/it/BeanioTest.java

```diff
@@ -33,7 +33,7 @@
 import org.junit.jupiter.params.provider.ValueSource;
 
 import static org.apache.camel.quarkus.component.beanio.it.BeanioResource.FORMATTER;
-import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.*;
```

Comment:
Can we avoid `*` imports. There's only one additional import required for this change.

Reply:
I already configured my ide several times to do not use .*. I'll do that once again

Reply:
fixed